deb/publish/publish.sh

```diff
@@ -36,11 +36,11 @@ cat $D/merged/Contents | gzip -9c > $D/binary/Contents.gz
 apt-ftparchive -c $bin/release.conf release $D/binary > $D/binary/Release
 # sign the release file
 rm $D/binary/Release.gpg || true
-gpg --batch --no-use-agent --no-default-keyring --keyring $GPG_KEYRING --secret-keyring=$GPG_SECRET_KEYRING --passphrase-file $GPG_PASSPHRASE_FILE \
+gpg --batch --no-use-agent --no-default-keyring --keyring "$GPG_KEYRING" --secret-keyring="$GPG_SECRET_KEYRING" --passphrase-file "$GPG_PASSPHRASE_FILE" \
```

Comment:
same thing in bash... - but you logically want to quote the LHS of the equal so I find this more readable as to why this is quoted.